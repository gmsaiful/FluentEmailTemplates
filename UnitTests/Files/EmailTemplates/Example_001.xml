<emailTemplate xmlns="http://visualproduct.com/emailtemplate.xsd">
    <subject>
        <!-- The subject can contain merge fields. -->
        <value>Example 001 subject for *|FirstName|* *|LastName|*</value>
    </subject>
    <htmlBody>
        <!-- Include the standard email header html file. Relative path is appended to the app setting "FluentEmailTemplatesFilePath". -->
        <htmlFile relativePath="EmailContent\StandardEmailHeader.html" />
        
        <!-- Html template with merge fields. -->
        <html>
            <table style="width:600px;margin-top:20px;margin-bottom:20px;">
                <tr>
                    <td style="padding:10px 10px 10px 10px;">
                        Hi *|FirstName|*,
                            
                        <br /><br />
                        This is example email template 001 with some basic merge fields. Nothing too fancy.
                                
                        <br /><br />
                        <p style="font-size:14px;font-weight:bold;">Your address details:</p>
                        <br />*|Address1|*
                        <br />*|Address2|*
                        <br />*|City|* *|PostCode|*
                        <br />*|Country|*
                            
                        <br /><br />
                        <p style="font-size:14px;font-weight:bold;">Thank you for your enquiry.</p>
                            
                        <br /><br />
                        Daffy Duck.
                        <br />Email: <a href="mailto:daffy.duck@example.com">daffy.duck@example.com</a>
                        <br /><a href="https://example.com">https://example.com</a>
                    </td>
                </tr>
            </table>
        </html>
        
        <!-- Include the standard email footer html file. Relative path is appended to the app setting "FluentEmailTemplatesFilePath". -->
        <htmlFile relativePath="EmailContent\StandardEmailFooter.html" />
    </htmlBody>
</emailTemplate>
