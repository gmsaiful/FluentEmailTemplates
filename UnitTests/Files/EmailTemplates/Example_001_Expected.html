<html>
    <body style="width:100%;color:#555555;padding:0 0 0 0;font-family:gotham, helvetica, arial, sans-serif;background-color:#CCCCCC;text-align:center;">
        <div style="width:620px;padding:10px 0 10px 0;margin:auto;text-align:left;">
            <table style="width:600px;background-color:white;">
                <tbody>
                    <tr>
                        <td><img src="https://example.com/images/email/logo.png" style="width:600px;height:100px;max-height:100px;" alt="example.com logo" /></td>
                    </tr>
                    <tr>
                        <td>
                            <!-- Start of content -->

            <table style="width:600px;margin-top:20px;margin-bottom:20px;" xmlns="http://visualproduct.com/emailtemplate.xsd">
                <tr>
                    <td style="padding:10px 10px 10px 10px;">
                        Hi Goofy,
                            
                        <br /><br />
                        This is example email template 001 with some basic merge fields. Nothing too fancy yet.
                                
                        <br /><br />
                        <p style="font-size:14px;font-weight:bold;">Your address details:</p>
                        <br />1 Goof Street
                        <br />The Kennel
                        <br />Nuke Town 007
                        <br />New Zealand
                            
                        <br /><br />
                        <p style="font-size:14px;font-weight:bold;">Thank you for your enquiry.</p>
                            
                        <br /><br />
                        Daffy Duck.
                        <br />Email: <a href="mailto:daffy.duck@example.com">daffy.duck@example.com</a>
                        <br /><a href="https://example.com">https://example.com</a>
                    </td>
                </tr>
            </table>
        
                        <!-- End of content -->
                        </td>
                    </tr>
                    <tr>
                        <td style="padding:10px 20px 10px 20px;border-top:solid 1px #CCCCCC;">
                            <div style="font-family:Geneva,Verdana,Arial,Helvetica,sans-serif;font-size:9px;line-height:1.34em;color:#999999;">
                                SUBSCRIPTION INFORMATION: You have received this message because you have indicated that you are interested
                                in receiving information from example.com. If you do not wish to receive further information simply go to
                                your <a href="https://example.com/account/notifications" style="color: #9999FF;">notifications</a>
                                page to change your subscription settings.
                            </div>
                        </td>
                    </tr>
                </tbody>
            </table>
        </div>
    </body>
</html>